styles/blocks/footer/footer.less

```
@@ -0,0 +1,106 @@
+.footer {
+    background-color: #212122;
+
+    &__social {
+        border: 1px solid transparent;
+
+        @media @screen-gt-s {
+            & {
+                float: left;
+                width: 50%;
+            }
+        }
+        @media @screen-gt-m {
+            & {
+                width: 33.3%;
+            }
+        }        
+    }
+
+    &__stores {
+        @media @screen-gt-s {
+            & {
+                margin-top: 30px;                
+                float: right;
+                width: 50%;
+                text-align:right;
+            }
+        }
+        @media @screen-gt-m {
+            & {
+                margin-top: 30px;                
+                float: left;
+                width: 33.3%;
+                text-align:right;
+            }
+        }        
+    }
+
+}
+
+.footer__logo{
+    display:none;
+
+    @media @screen-gt-m {
+        & {
+            display: block;
+            width: 33.3%;
+            float: right;
+            text-align: right;
+            margin-top: 43px;
+            margin-bottom:40px;
+            filter: invert() grayscale();
+
+            &_href{
+                transition: all 0.5s ease-in;                
+
+                &:hover{
+                    filter: invert() grayscale();
+                    
+                }
+            }
+
+        }
+    }
+}
+
+
+.footer__separator {
+    width:100%;
+    border-top: 1px #666 solid;
+    margin-top: 30px;
+    margin-bottom: 15px;
+    clear:both;
+
+    @media @screen-gt-m {
+        & {
+            margin-bottom: 30px;            /* Чтобы текст переносился как в макете  */
+        }
+    }        
+}
+
+.footer__copyright {
+    font-family: 'Arial';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
```

```suggestion
    font-family: 'Arial';
    font-style: normal;
    font-weight: 400;
    font-size: 14px;
    line-height: 20px;
```

эти стили наследуются от body, можно их не прописывать